integration/creategetdelete_test.go

```diff
@@ -109,6 +121,34 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 			})
 		})
 
+		Context("and configuring Flux for GitOps", func() {
+			It("should not return an error", func() {
+				// Use a random branch to ensure test runs don't step on each others.
+				branch := random.String(8)
```

Comment:
I'd just import `github.com/kubicorn/kubicorn/pkg/namer` and call `namer.RandomName()`, we use it in `pkg/ctl/cmdutils/namers.go` already, but here it'd be fine to just import and call it directly.

Reply:
Thanks. Done.